cheroot/test/test_conn.py

```diff
@@ -1000,3 +1041,73 @@ def test_No_CRLF(test_client, invalid_terminator):
     expected_resp_body = b'HTTP requires CRLF terminators'
     assert actual_resp_body == expected_resp_body
     conn.close()
+
+
+def test_invalid_selected_connection(test_client, monkeypatch):  # noqa: D200,D205,D400,E501
+    """Test the error handling segment of
+    ``cheroot.connections.ConnectionManager.get_conn``.
```

Comment:
I'm trying to maintain as low number of noqa as possible across the project:
```suggestion
def test_invalid_selected_connection(test_client, monkeypatch):
    """Test the error handling segment of HTTP connection selection.

    See ``cheroot.connections.ConnectionManager.get_conn``.
```